src/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Please kindly implement a method to add rentals as stated in the requirement, kindly check this [link for better understanding](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). 🙏 

<img width="554" alt="g126" src="https://github.com/alexansaa/School-Library/assets/103386183/d25ad3f1-d548-45d7-b6b9-fbb9b4201445">